apps/asap-server/src/controllers/groups.ts

```diff
@@ -15,9 +15,8 @@ flatData {
   summary
```

Comment:
This should be deleted from the query if we are deleting summary also in this PR

Reply:
This needs to be reflected on the entity and fixture files